docs/src/Accessibility.doc.js

```diff
@@ -0,0 +1,250 @@
+
+      - In [AvatarGroup](/AvatarGroup#ARIA-attributes), \`accessibilityLabel\` is used to describe an interactive Avatar face-pile that also acts as a button. A good \`accessibilityLabel\` may be "Collaborators: Keerthi, Alberto, and 10 more. Add collaborators to this board."
+      - In [Callout](/Callout#Accessibility) and [Upsell](/Upsell#Accessibility), \`accessibilityLabel\` can be specified for each primary and secondary action to create better context. If the button texts are "Learn More" and "Invite", the accessibility labels should be "Learn more about ads credit" and "Invite friends for ads credit". If the button text is already descriptive enough, the accessibilityLabel can be an empty string.
+    `}
+    />
+    <MainSection.Subsection
+      title="Keyboard navigation"
+      description={`
+        People use and interact with technology in a myriad of ways. Many people do not use a mouse for various reasons and instead rely on the keyboard and a [screen reader](https://www.afb.org/blindness-and-low-vision/using-technology/assistive-technology-products/screen-readers) to interact with applications. The key to creating truly accessible products is ensuring that your application works very well when using only a keyboard and/or screen reader to navigate and interact with content. In other words, if it can be done with a mouse, it can be done with a keyboard. In the Gestalt documentation, we aim to help describe the expectations for keyboard navigation, particularly within our more complex components. For example, in [ComboBox](/ComboBox#Keyboard-interaction), we've detailed the expected keyboard interaction for all aspects of the ComboBox.
+
+        Another key aspect to keyboard navigation is [focus management](https://css-tricks.com/focus-management-and-inert/). Users should never lose their place within a page, and their current point of focus should always be clear visually. Some components, like Modals or Popovers, act as focus traps, meaning the user's focus should only rotate between items inside the Modal or Popover, and anything below these items is not reachable by keyboard. When a component like Modal or Popover is dismissed or closed, the user's focus should go back to the item that triggered the component.
+    `}
+    />
+    <MainSection.Subsection
+      columns={2}
+      title="Available hooks"
+      description={`Gestalt provides two functions that help create more accessible experiences: \`useFocusVisible\` and \`useReducedMotion\`.`}
```

Comment:
We can add external links here, right? Consider replacing "functions" with "custom Hooks" and link that to the React docs on custom Hooks